zerver/lib/realm_night_logo.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+
+from zerver.lib.upload import upload_backend
+from zerver.models import Realm
+
+def realm_night_logo_url(realm: Realm) -> str:
+    return get_realm_night_logo_url(realm)
+
+def get_realm_night_logo_url(realm: Realm) -> str:
+    if realm.night_logo_source == 'U':
+        return upload_backend.get_realm_night_logo_url(realm.id, realm.night_logo_version)
+    else:
+        return settings.DEFAULT_LOGO_URI+'?version=0'
```

Comment:
Let's move this stuff into `realm_logo.py` -- don't need a separate file.  And also see my comments about making these functions just take a `night` bool as an argument.